apis/conversion/src/export-format.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Used for exporting and importing properties between backend models. An object that exports to
+ * ExportFormat should put dimensions and measures inside one data group. If an object has two hypercubes
+ * each of the cubes should export dimensions and measures in two separate data groups.
+ * An object that imports from this structure is responsible for putting the existing properties where they should be
+ * in the new model.
+ *
+ * @param nDataGroups The number of hypercubes.
```

Comment:
Should we add the @ingore tag here maybe? I'm thinking it's an "internal" function...

Reply:
This is used as a parameter for importPropeties and as a return for exportProperties. I will update the doc for ExportFormat

Reply:
it should be ignored until we release it, but also need to make sure the specs work properly. So maybe add in a label of sorts for now DO_NOT_USE so it shows up in the specs, just in case we do an early release.